src/macros.rs

```diff
@@ -21,14 +21,24 @@ macro_rules! algos {
 
     ($($mod:ident)::+<$inner:ident>) => {
         algos!(@algo brotli ["brotli"] BrotliDecoder BrotliEncoder<$inner> {
-            /// The `level` argument here is typically 0-11.
-            pub fn new(reader: $inner, level: u32) -> Self {
-                let mut params = brotli::enc::backward_references::BrotliEncoderParams::default();
-                params.quality = level as _;
-                Self::from_params(reader, params)
+            pub fn new(reader: $inner) -> Self {
+                let params = brotli::enc::backward_references::BrotliEncoderParams::default();
```

Comment:
You could replace the body of `new` with `Self::with_quality(reader, crate::Compression::Default)`, that could also let you move it to the above macro branch to avoid repeating it for each algorithm

```rust
            $encoder<$inner> {
                pub fn new(inner: $inner) -> Self {
                    Self::with_quality(inner, crate::Compression::Default)
                }
            } $({ $($constructor)* })*
```

Reply:
That's great, done!